xml/System/Memory`1.xml

```diff
@@ -66,11 +99,24 @@
         <Parameter Name="length" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="array">To be added.</param>
-        <param name="start">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="array">The source array.</param>
+        <param name="start">The index of the first element to include in the new <see cref="T:System.Memory`1" />.</param>
+        <param name="length">The number of elements to include in the new <see cref="T:System.Memory`1" />.</param>
+        <summary>Creates a new <see cref="T:System.Memory`1" /> object that includes a specified number of elements of an array beginning at a specified index. </summary>
+       <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+If `array` is `null`, this constructor returns a `null` <xref:System.Memory%601> object.
```

Comment:
As per Ahson's comments earlier, you don't get back `null`... you get back `default(Memory<T>)`, i.e. basically a zero'd-out `Memory<T>`.